runner/runner.go

```diff
@@ -138,20 +138,23 @@ func (r *Runner) Run(ctx context.Context, query ...string) error {
 		return err
 	}
 
-	writerName := stdoutWriterName
-	outputWriter.AddWriters(NamedWriter{os.Stdout, stdoutWriterName})
+	var writerName string
```

Comment:
It seems like that the named writers introduced in https://github.com/projectdiscovery/uncover/pull/73 adds too much complexity. Let's remove them and simplify OutputWriter to:
```go
type OutputWriter struct {
	cache   *lru.Cache
	writers []io.Writer
	sync.RWMutex
}
```

Reply:
Done